core/templates/dev/head/pages/admin-page/admin-page.directive.ts

```diff
@@ -16,14 +16,10 @@
  * @fileoverview Data and directive for the Oppia admin page.
  */
 
-// TODO(vojtechjelinek): this block of requires should be removed after we
-// introduce webpack for /extensions
```

Comment:
Only remove the comments where there are no remaining files.

Reply:
I think the files used here are actually needed by the admin-page.directive